controllers/oauth.js

```diff
@@ -98,7 +97,7 @@ async function confirmed(req, res, next, client = FxAOAuthClient) {
         date: HBSHelpers.prettyDate(req.supportedLocales, new Date()),
         unsafeBreachesForEmail: unsafeBreachesForEmail,
         scanAnotherEmailHref: EmailUtils.getScanAnotherEmailUrl(utmID),
-        unsubscribeUrl: unsubscribeUrl,
+        unsubscribeUrl: EmailUtils.getUnsubscribeUrl(verifiedSubscriber, utmID),
```

Comment:
Note: this addresses https://github.com/mozilla/blurts-server/issues/763 but doesn't add any content to the unsubscribe page to tell the user that unsubscribing from Monitor alerts will leave their Firefox Account intact.

Reply:
> doesn't add any content to the unsubscribe page to tell the user that unsubscribing from Monitor alerts will leave their Firefox Account intact.

Do we need a new bug for that?

Reply:
Filed https://github.com/mozilla/blurts-server/issues/773